### DISCUSSION TITLE
Why are badges not displaying logos on GitHub Mobile?

### DISCUSSION THREAD

**Discussion by Andre601** [badges/shields#6337] [Q&A] [answered]
When using GitHub's App (In my case for Android) will you notice that badges which usually depict a logo on the left side have a blank area instead. Is this intentional?

Here's how the badges look on Desktop (Using Edge Browser):
![image](https://user-images.githubusercontent.com/11576465/113135255-31fade80-9222-11eb-80ef-18b770dbbadd.png)

And here are the same badges but viewed on mobile with the GitHub App:
![image](https://cdn.discordapp.com/attachments/584040473186926623/826775133079076894/SmartSelect_20210331-130856_GitHub.jpg)

Any info on why this happens or if that is even a bug would be appreciated.

**Comment by chris48s** [comment, marked as answer]
Today I learned GitHub has an android app :laughing:  Never used it before, but I just tried installing it and I get the same behaviour as you on shields.io badges with logos. It is definitely not intentional.

Interestingly, some ( non-shields.io ) badges that use logos do render the logo (although maybe it is part of the SVG not a `xlink:href`) and there is at least one other display glitch that I've spotted so far

browser:
![Screenshot at 2021-04-01 19-40-40](https://user-images.githubusercontent.com/6025893/113339863-03b0f800-9323-11eb-89bb-7adbc19c8e5a.png)

app:
![Screenshot_20210401-193751](https://user-images.githubusercontent.com/6025893/113340612-0829e080-9324-11eb-8d2d-db327ec9b489.png)


I suspect some of this might be about the SVG rendering in their app, but I'll have a bit more of an investigate and see if there is anything we can do on our side. This will probably be a pain to investigate as mobile apps are basically a black box :(

**Reply by chris48s** [reply]
Not to self, as much as anything.. Github's app uses https://github.com/BigBadaboom/androidsvg internally to render SVGs.
